recipes/iot-services/iotagent-lwm2m/config.js

```diff
@@ -0,0 +1,95 @@
+var config = {};
+
+config.lwm2m = {
+    logLevel: 'DEBUG',
+    port: 5684,
+    defaultType: 'Device',
+    ipProtocol: 'udp4',
+    serverProtocol: 'udp4',
+    formats: [
+        {
+            name: 'application-vnd-oma-lwm2m/text',
+            value: 1541
+        },
+        {
+            name: 'application-vnd-oma-lwm2m/tlv',
+            value: 1542
+        },
+        {
+            name: 'application-vnd-oma-lwm2m/json',
+            value: 1543
+        },
+        {
+            name: 'application-vnd-oma-lwm2m/opaque',
+            value: 1544
+        }
+    ],
+    writeFormat: 'application-vnd-oma-lwm2m/text',
+    types: [ 
+        {
+            name: 'Robot',
+            url: '/robots'
+        }
+    ]
+};
+
+config.ngsi = {
+    logLevel: 'DEBUG',
+    contextBroker: {
+        host: 'orion',
+        port: '1026'
+    },
+    server: {
+        port: 4041
+    },
+    deviceRegistry: {
+        type: 'memory'
+    },
```

Comment:
why not mongo?

Reply:
It was just the default in the original repo. Updated to use mongo by default.